lib/types/jpg.js

```diff
@@ -53,7 +53,7 @@ function calculate (buffer) {
     buffer = buffer.slice(i + 2);
   }
 
-  throw new TypeError('Invalid JPG, no size found');
+  throw new TypeError('Invalid JPG, no size found: ' + filepath);
```

Comment:
this will change the error message for everyone else who's using buffers.. why not catch the error in your code & log the file name?
not to mention that most bug trackers group by the error message. This change will break those groupings.